datafusion/physical-plan/src/stream.rs

```diff
@@ -110,7 +211,7 @@ impl RecordBatchReceiverStreamBuilder {
     ) {
         let output = self.tx();
 
-        self.spawn(async move {
+        self.inner.spawn(async move {
```

Comment:
This logic by comparison is rather ExecutionPlan specifi, and I don't think valuable to DRY